src/BlazorWebView/src/SharedSource/WebView2WebViewManager.cs

```diff
@@ -46,17 +46,51 @@ public class WebView2WebViewManager : WebViewManager
 		// making it substantially faster. Note that this isn't real HTTP traffic, since
 		// we intercept all the requests within this origin.
 		internal static readonly string AppHostAddress = "0.0.0.0";
+
+		/// <summary>
+		/// Gets the application's base URI. Defaults to <c>https://0.0.0.0/</c>
+		/// </summary>
 		protected static readonly string AppOrigin = $"https://{AppHostAddress}/";
 
 		private readonly WebView2Control _webview;
 		private readonly Task _webviewReadyTask;
+
 #if WEBVIEW2_WINFORMS || WEBVIEW2_WPF
 		private protected CoreWebView2Environment _coreWebView2Environment;
 		private readonly Action<ExternalLinkNavigationEventArgs> _externalNavigationStarting;
+
+		/// <summary>
```

Comment:
@TanayParikh turns out you can't use ifdefs in the middle of xml docs. Replicating the ctors seemed a lot cleaner than what we previously had. Thoughts?

Reply:
Looks good to me!